reproduction.py

```diff
@@ -0,0 +1,163 @@
+
+
+pieces_path = {
+    "P": "plt",
+    "B": "blt",
+    "N": "nlt",
+    "R": "rlt",
+    "Q": "qlt",
+    "K": "klt",
+    "p": "pdt",
+    "b": "bdt",
+    "n": "ndt",
+    "r": "rdt",
+    "q": "qdt",
+    "k": "kdt",
+}
+empty_img = "img/60px_empty.png"
+pieces_name = "img/60px-Chess_{}.png"
+empty_img_30 = "img/30px_empty.png"
+pieces_name_30 = "img/30px-Chess_{}.png"
+
+html = '''<!DOCTYPE html>
+<html>
+    <head>
+        <title></title>
+        <meta charset="UTF-8">
+        <style>
+            .chess-board { border-spacing: 0; border-collapse: collapse; margin-bottom: -1em; }
+            .chess-board th { padding: .5em; }
+            .chess-board td { border: 1px solid; width: 60px; height: 60px; }
+            .chess-board .light { background: whitesmoke; }
+            .chess-board .dark { background: sienna }
+            .small-table { border-spacing: 0px; border-collapse: none; border-radius: 0px; margin-bottom: -1em; }
+            .small-table td { border: 0px none; width: 30px; height: 30px; }
+            .small-table tr { border: 0px none; width: 30px; height: 30px; }
+            .small-table .light { background: whitesmoke; }
+            .small-table .dark { background: sienna }
+        </style>
+    </head>
+    <body>
+        <table>
+            <tbody>
+                <tr>
+                    <td style="text-align: center">
+                        <h1> Original: </h1>
+                        <table class="chess-board">
+                            <tbody>
+                                %s
+                            </tbody>
+                        </table>
+                        <h2 style="margin-bottom: 0em">castling: %s </h2>
+                    </td>
+                    <td style="text-align: center">
+                        <h1> Reproducted: </h1>
+                        <table class="chess-board">
+                            <tbody>
+                                %s
+                            </tbody>
+                        </table>
+                        <h2 style="margin-bottom: 0em">castling: %s </h2>
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+    </body>
+</html>'''
+
+small_table_temp = '''
+    <table class="small-table">
+    <tr>
+      <td class="{0}"><img src="{1[1]}" style="opacity:{1[0]}; height: 100%"></td>
+      <td class="{0}"><img src="{2[1]}" style="opacity:{2[0]}; height: 100%"></td>
+    </tr>
+    <tr>
+      <td class="{0}"><img src="{3[1]}" style="opacity:{3[0]}; height: 100%"></td>
+      <td class="{0}"><img src="{4[1]}" style="opacity:{4[0]}; height: 100%"></td>
+    </tr>
+    </table>'''
+
+
+def display_fen(fen):
+    fen = fen.split(' ')
+    board_fen = fen[0]
+    player = fen[1]
+    table = display_board(board_fen, player == 'w')
+    castling = fen[2].replace('K', '♔').replace('Q', '♕').replace('k', '♚').replace('q', '♛')
+    return table, castling
+
+
+def display_board(fen, player):
```

Comment:
this one converts to HTML, doesn't display, so better name should be "board_to_html"